charts/weblate/templates/statefulset.yaml

```diff
@@ -0,0 +1,121 @@
+            - name: POSTGRES_USER
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "weblate.fullname" . }}
+                  key: postgres_user
+            - name: POSTGRES_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "weblate.fullname" . }}
+                  key: postgres_password
+            - name: POSTGRES_DATABASE
+              value: {{ .Values.postgres.database | quote }}
+            - name: REDIS_HOST
+              {{- if and .Values.redis.enabled }}
+              value: {{ include "weblate.fullname" . }}-redis
+              {{- else }}
+              value: {{ .Values.redis.host | quote }}
+              {{- end }}
+            - name: WEBLATE_EMAIL_HOST
+              value: {{ .Values.email.host | quote}}
+            - name: WEBLATE_EMAIL_HOST_USER
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "weblate.fullname" . }}
+                  key: email_user
+            - name: WEBLATE_EMAIL_HOST_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "weblate.fullname" . }}
+                  key: email_password
+            - name: WEBLATE_SERVER_EMAIL
+              value: {{ .Values.email.from | quote }}
+            - name: WEBLATE_DEFAULT_FROM_EMAIL
+              value: {{ .Values.email.from | quote }}
+            {{- range $key,$value := .Values.env }}
+            - name: {{ $key | quote }}
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "weblate.fullname" $ }}
+                  key: {{ $key | lower | quote }}
+            {{- end }}
+          ports:
+            - name: http
+              containerPort: 8080
+              protocol: TCP
+          livenessProbe:
+            httpGet:
+              path: /healthz/
+              port: http
+            initialDelaySeconds: 30
+            timeoutSeconds: 30
+            periodSeconds: 30
+          readinessProbe:
+            httpGet:
+              path: /healthz/
+              port: http
+            initialDelaySeconds: 30
+            timeoutSeconds: 30
+            periodSeconds: 30
+          resources:
+            {{- toYaml .Values.resources | nindent 12 }}
+          volumeMounts:
+            - name: data
+              mountPath: /app/data
+      {{- with .Values.nodeSelector }}
+      nodeSelector:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+    {{- with .Values.affinity }}
+      affinity:
+        {{- toYaml . | nindent 8 }}
+    {{- end }}
+    {{- with .Values.tolerations }}
+      tolerations:
+        {{- toYaml . | nindent 8 }}
+    {{- end }}
+      securityContext:
+        fsGroup: 1000
+      volumes:
+        - name: data
+          {{- if .Values.persistence.enabled }}
+          persistentVolumeClaim:
+            claimName: {{ include "weblate.fullname" . }}
```

Comment:
I know weblate probably doesn't support horizontal scaling at the moment but for StatefulSets it's more natural to [use `volumeClaimTemplates`](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#components). What do you think?

Reply:
I've never used `volumeClaimTemplates` - so ill have to have a look.